### PR TITLE
[14.0] Switch tests to GH actions

### PR DESCRIPTION
**Pull request by simahawk**

### CODE REVIEW

**Comment by simahawk** [comment]
@sbidoul thanks for manifestoo! We didn't know about this problem :handshake: 

image

**Comment by sbidoul** [comment]
Yep... wait until you run it on your private repos :)

**Comment by sebalix** [comment]
Nice! We should set the AGPL-3 on `stock_reception_screen_qty_by_packaging` then.

**Review by sebalix — APPROVED** [review]
Pre-approving

**Review by mmequignon — APPROVED** [review]
pre-approving as well

**Comment by simahawk** [comment]
base_jsonify https://github.com/OCA/server-tools/pull/2257
stock_reception_screen_qty_by_packaging will be changed in this PR

File: .t2d.yml [hunk 1]
```diff
@@ -12,8 +12,8 @@ addons:
   apt:
     packages:
       - expect-dev # provides unbuffer utility
-      - cups
-      - libcups2-dev
+      - "cups"
+      - "libcups2-dev"
```
Comment (sbidoul) [inline .t2d.yml:16]: Note

These are part of the [oca/oca-ci Dockerfile](https://github.com/OCA/oca-ci/blob/9eff1d7235a52bde738b6ed1503e766a38f6dd70/Dockerfile#L71), as it is the only place we can declare *build* dependencies for third party python libs that don't publish manylinux wheels.

*runtime* deb dependencies, OTOH have to be declared in manifests, in a `deb` key of the `external_dependencies` section.

Comment (simahawk) [reply .t2d.yml:16]: @sbidoul it comes from a dependency on `base_report_to_printer` but that module declares only `pycups` as a dep indeed https://github.com/OCA/report-print-send/blob/14.0/base_report_to_printer/__manifest__.py#L32
Shall I update that?

**Comment by simahawk** [comment]
https://github.com/OCA/stock-logistics-warehouse/pull/1352
https://github.com/OCA/wms/pull/361

**Comment by simahawk** [comment]
https://github.com/OCA/stock-logistics-workflow/pull/955

**Comment by simahawk** [comment]
https://github.com/OCA/product-attribute/pull/988

**Comment by simahawk** [comment]
:green_circle: :man_dancing:

**Pull request merged**